test/integration/http_protocol_integration.cc

```diff
@@ -113,9 +115,11 @@ std::string HttpProtocolIntegrationTest::protocolTestParamsToString(
                       http2ImplementationToString(params.param.http2_implementation),
                       params.param.defer_processing_backedup_streams ? "WithDeferredProcessing"
                                                                      : "NoDeferredProcessing",
+                      params.param.use_universal_header_validator ? "Uhv" : "Legacy",
                       params.param.deprecate_callback_visitor ? "WithCallbackVisitor"
                                                               : "NoCallbackVisitor",
-                      params.param.use_universal_header_validator ? "Uhv" : "Legacy");
+                      params.param.tunneling_with_upstream_filters ? "WithUpstreamHttpFilters"
```

Comment:
I think this is the probem.  Let's only do this for tests using TCP proxying. 

/wait